docs/about/whats_new.rst

```diff
@@ -10,6 +10,7 @@ v1.9.next
 
 - Fix typos in docs (:pull:`1577`)
 - Merge in 1.8.x branch changes. (:pull:`1568`, :pull:`1579`)
+- Add Product delete methods to API and command in CLI, plus misc cleanup of the surrounds (:pull:`1583`)
```

Comment:
Feel free to mention here that user requests for this feature have been around longer than the name of the project.